src/digraph.c

```diff
@@ -40,62 +40,62 @@ static digr_T digraphdefault[] =
 	 * ATARI digraphs
 	 */
     {{'C', ',', 128},  /* ~@ XX */
-     {'u', '"', 129},  /* � */
```

Comment:
Well, that broke and I never noticed...

Reply:
It's only comments though? Maybe we can put UTF-8 in its place?

Reply:
Ah, interesting. I think replacing with UTF-8 would be reasonable.